reference/release-notes/1.23.1.md

```diff
@@ -0,0 +1,76 @@
+---
+orphan: true
+---
+# 1.23.1
+
+These release notes cover new features and changes in Anbox Cloud 1.23.1.
+
+Anbox Cloud 1.23.1 is a patch release. To understand minor and patch releases, see {ref}`ref-release-notes`.
+
+Please see {ref}`ref-component-versions` for a list of updated components.
+
+## Requirements
+
+See {ref}`ref-requirements` for detailed requirements to run Anbox Cloud 1.23.1.
+
+## New features & improvements
+
+### Anbox Cloud Appliance (epoch=1)
+
+* You can now change the appliance network configuration using `anbox-cloud-appliance config set`. You can change the public IP address and location of the appliance.<!--AC-2727-->.
+* You can now configure the [CORS settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for those API endpoints that the appliance reverse proxy passes to the Anbox Stream Gateway. You can set the following using the `anbox-cloud-appliance config set` command:<!--AC-2732-->
+  - The network's public IP address (`network.public_address`)
+  - The network's DNS name (`network.location`)
```

Comment:
These two belong to the item above (`You can now change the appliance network ...`)

Reply:
Thanks, both of them being the same PR confused me :)